src/main/app/common/claimDocumentType.ts

```diff
@@ -31,4 +31,8 @@ export class ClaimDocumentType {
     text: 'Download the request to review the order',
     uri: 'review-order'
   }
+  static readonly GENERAL_LETTER = {
+    text: 'Download the general letter',
+    uri: 'GENERAL_LETTER'
```

Comment:
other document can be uniquely identified with this uri, but problem with this general letter is that there can be multiple general letters and so they have to be uniquely identified so id has to be added to be sent to backend. So while appending here itself, it made GENRAL Letter in uppercase, otherwise I will do same thing in document.ts by applying extra if else conditions or in backend. So somewhere this has to be done. This uppercase is then converted to ENUM in backend